content/tutorial/tutorial.md

```diff
@@ -739,7 +739,7 @@ We can now change the Board's `handleClick` function to return early by ignoring
 ```javascript{3-5}
   handleClick(i) {
     const squares = this.state.squares.slice();
-    if (calculateWinner(squares) || squares[i]) {
```

Comment:
This change is to avoid call to calculateWinner function if game is not finished